src/routes/home/components/HomeScreenHeader.tsx

```diff
@@ -41,34 +45,47 @@ type HomeScreenHeaderProps = {
 }
 
 const HomeScreenHeader = ({ navigation }: HomeScreenHeaderProps): JSX.Element => {
-  const theme = useTheme()
+  const [overlayVisible, setOverlayVisible] = useState(false)
+
+  useFocusEffect(useCallback(() => setOverlayVisible(false), []))
+
+  if (overlayVisible) {
+    return (
+      <OverlayMenu setVisible={setOverlayVisible}>
+        <OverlayMenuItem
+          title={getLabels().general.header.manageSelection}
+          onPress={() => navigation.navigate('ManageSelection')}
+        />
+        <OverlayMenuSeparator />
+        <OverlayMenuItem
+          isSubItem
+          title={getLabels().general.header.sponsors}
+          onPress={() => navigation.navigate('Sponsors')}
+        />
+        <OverlayMenuItem
+          isSubItem
+          title={getLabels().general.header.settings}
+          onPress={() => navigation.navigate('Settings')}
+        />
+        <OverlayMenuItem
+          isSubItem
+          title={getLabels().general.header.impressum}
+          onPress={() => navigation.navigate('Imprint')}
+        />
+      </OverlayMenu>
+    )
+  }
+
   return (
     <Wrapper testID='header'>
       <HeaderStyle>
         <SmileIconStyle>
           <LunesIcon width={hp('8%')} height={hp('8%')} />
         </SmileIconStyle>
         <HeaderButtonsContainer>
-          <OverflowMenu
-            icon={
-              <MenuIconContainer>
-                <MenuIconWhite width={theme.spacingsPlain.md} height={theme.spacingsPlain.md} />
-              </MenuIconContainer>
-            }>
-            <HiddenItem
-              titleStyle={{ fontSize: hp('2%') }}
-              title={getLabels().general.header.manageSelection}
-              onPress={() => navigation.navigate('ManageSelection')}
-              // iOS has issue using styled(HiddenItem)
-              style={{ height: theme.spacingsPlain.xl }}
-            />
-            <HiddenItem
-              title={getLabels().general.header.settings}
-              onPress={() => navigation.navigate('Settings')}
-              titleStyle={{ fontSize: hp('2%') }}
-              style={{ height: theme.spacingsPlain.xl }}
-            />
-          </OverflowMenu>
+          <MenuIconContainer>
+            <MenuIconWhite testID='menu-icon-white' onPress={() => setOverlayVisible(true)} />
```

Comment:
The button to press here is tiny, it's really only the three dots of the icon, whereas previously it was a normal, square area around the icon. At least on iOS emulator, it's pretty hard to press the right spot.